corehq/apps/app_manager/app_translations/generators.py

```diff
@@ -190,21 +192,30 @@ def _metadata(self):
 
     def generate_translation_files(self):
         self._build_translations()
```

Comment:
Just pointing out that this object has the same pattern that `PoFileGenerator` had before acdf4f4812b58c9196f9ffe7c091af4cff145dcb: it's in a partially initialized state until `generate_translation_files()` is called. I'm not going to ask you to re-architect it in this PR, but it's some food for thought.